### PR TITLE
(phi0134)_all files

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Deleted Latin name from textgroup with up to date practice, but left the
full Latin name for author in header and bibl descriptions.

Updated cts_.xml files
Updated the bibls and added in page links to online files.

### CODE REVIEW

File: data/phi0134/phi004/phi0134.phi004.perseus-eng2.xml [hunk 1]
```diff
@@ -2,9 +2,9 @@
    <teiHeader>
       <fileDesc>
          <titleStmt>
-            <title>Phormio, or The Scheming Parasite</title>
-            <author>P. Terentius Afer (Terence)</author>
-            <editor role="transl">Henry Thomas Riley</editor>
+            <title>The Scheming Parasite</title>
```
Comment (lcerrato) [inline data/phi0134/phi004/phi0134.phi004.perseus-eng2.xml:5]: @AlisonBabeu 
I have a question about the English title modifications as I think that the print does indeed read "Phormio, or The Scheming Parasite" so I wouldn't make this change. This probably applies elsewhere?

**Review by lcerrato — CHANGES_REQUESTED** [review]
Recheck the English titles used as they should be as in print.

File: data/phi0134/phi004/phi0134.phi004.perseus-eng2.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/phi0134/phi004/phi0134.phi004.perseus-eng2.xml:5]: Ok, I wasn't sure. I will change it back so it reads exactly as in the printed volume for the header.

Comment (lcerrato) [reply data/phi0134/phi004/phi0134.phi004.perseus-eng2.xml:5]: Yes, I think those titles for all of that set of translations are as the print reads. I know they look like something we might have added, but I don't think that's true. The work files should be consistent with that, too.
(I only looked at Phormio).

Comment (AlisonBabeu) [reply data/phi0134/phi004/phi0134.phi004.perseus-eng2.xml:5]: I added in page level links and you are right that is how they show up in the printed volume. I will change it in the header and the cts work level

**Review by lcerrato — COMMENTED** [review]
@AlisonBabeu 
There are a few minor inconsistencies between the cts files and the titles. I'll take a look at the book after the meeting

File: data/phi0134/phi001/__cts__.xml [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-              <ti:label xml:lang="eng">Andria, Publii Terentii Comoediae sex</ti:label>
-              <ti:description xml:lang="eng">Terence, creator; Parry, Edward St. John, editor</ti:description>
+              <ti:label xml:lang="lat">Andria</ti:label>
+              <ti:description xml:lang="mul">Terence. Publii Terentii Comoediae Sex. Parry, Edward St. John, editor. London: Whittaker and 
+                Co.; George Bell, 1857.</ti:description>
             </ti:edition>
           <ti:translation xml:lang="eng" urn="urn:cts:latinLit:phi0134.phi001.perseus-eng2" workUrn="urn:cts:latinLit:phi0134.phi001">
-              <ti:label xml:lang="eng">Andria, The comedies of Terence</ti:label>
-              <ti:description xml:lang="eng">Terence, creator; Riley, Henry Thomas, 1816-1878, translator; Colman, George, 1732-1794, translator</ti:description>
+              <ti:label xml:lang="eng">Andria: The Fair Andrian</ti:label>
```
Comment (lcerrato) [inline data/phi0134/phi001/__cts__.xml:9]: semi colon instead of colon in title

File: data/phi0134/phi001/phi0134.phi001.perseus-eng2.xml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"	schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"	schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0">
-   <teiHeader>
+   <teiHeader xml:lang="eng">
       <fileDesc>
          <titleStmt>
-            <title>Andria:  The Fair Andrian</title>
-            <author>P. Terentius Afer (Terence)</author>
-            <editor role="transl">Henry Thomas Riley</editor>
+            <title>Andria: The Fair Andrian</title>
```
Comment (lcerrato) [inline data/phi0134/phi001/phi0134.phi001.perseus-eng2.xml:5]: semi colon instead of colon in title

File: data/phi0134/phi002/__cts__.xml [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
-              <ti:description xml:lang="eng">Terence, creator; Parry, Edward St. John, editor</ti:description>
+              <ti:label xml:lang="lat">Heautontimorumenos</ti:label>
+              <ti:description xml:lang="mul">Terence. Publii Terentii Comoediae Sex. Parry, Edward St. John, editor. London: Whittaker and 
+                Co.; George Bell, 1857.</ti:description>
             </ti:edition>
           <ti:translation xml:lang="eng" urn="urn:cts:latinLit:phi0134.phi002.perseus-eng2" workUrn="urn:cts:latinLit:phi0134.phi002">
-              <ti:label xml:lang="eng">The Self-Tormenter, The comedies of Terence</ti:label>
-              <ti:description xml:lang="eng">Terence, creator; Riley, Henry Thomas, 1816-1878, translator; Colman, George, 1732-1794, translator</ti:description>
-            </ti:translation>
+            <ti:label xml:lang="eng">Heautontimorumenos, The Self-Tormenter</ti:label>
```
Comment (lcerrato) [inline data/phi0134/phi002/__cts__.xml:9]: semi colon instead of comma in title

File: data/phi0134/phi002/phi0134.phi002.perseus-eng2.xml [hunk 5]
```diff
@@ -1,10 +1,10 @@
 <?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"	schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"	schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0">
-   <teiHeader>
+   <teiHeader xml:lang="eng">
       <fileDesc>
          <titleStmt>
-            <title>Heautontimorumenos: The Self-Tormenter</title>
-            <author>P. Terentius Afer (Terence)</author>
-            <editor role="transl">Henry Thomas Riley</editor>
+            <title>Heautontimorumenos, The Self-Tormenter</title>
```
Comment (lcerrato) [inline data/phi0134/phi002/phi0134.phi002.perseus-eng2.xml:5]: semi colon instead of comma in title

File: data/phi0134/phi003/__cts__.xml [hunk 6]
```diff
@@ -1,11 +1,13 @@
...
-              <ti:description xml:lang="eng">Terence, creator; Parry, Edward St. John, editor</ti:description>
+              <ti:label xml:lang="lat">Eunuchus</ti:label>
+              <ti:description xml:lang="mul">Terence. Publii Terentii Comoediae Sex. Parry, Edward St. John, editor. London: Whittaker and 
+                Co.; George Bell, 1857.</ti:description>
             </ti:edition>
           <ti:translation xml:lang="eng" urn="urn:cts:latinLit:phi0134.phi003.perseus-eng2" workUrn="urn:cts:latinLit:phi0134.phi003">
-              <ti:label xml:lang="eng">The Eunuch, The comedies of Terence</ti:label>
-              <ti:description xml:lang="eng">Terence, creator; Riley, Henry Thomas, 1816-1878, translator; Colman, George, 1732-1794, translator</ti:description>
-            </ti:translation>
+              <ti:label xml:lang="eng">Eunuchus, The Eunuch</ti:label>
```
Comment (lcerrato) [inline data/phi0134/phi003/__cts__.xml:9]: semi colon in title (translation file title is correct)

**Review by lcerrato — COMMENTED** [review]
review in progress

**Comment by AlisonBabeu** [comment]
Ok , sorry. I think I'm close!

**Review by lcerrato — APPROVED** [review]

**Pull request merged**